src/lib/Layout/PanelSideBarLayout/PanelSideBarLayout.tsx

```diff
@@ -0,0 +1,74 @@
+import classNames from "classnames";
+import { PropsWithChildren, ReactNode, useState } from "react";
+import { DropdownMenu, DropdownToggle, Nav, NavItem, UncontrolledDropdown } from "reactstrap";
+import "../../../../styles/Layout/PanelSideBarLayout.scss";
+import { PanelSideBar } from "./PanelSideBar/PanelSidebar";
+import { PanelSideBarLayoutContent } from "./PanelSideBarLayoutContent";
+import { PanelSideBarToggle } from "./PanelSideBar/PanelSideBarToggle";
+import {usePanelSideBarContext} from "src/lib/Layout/PanelSideBarLayout/PanelSideBar/Context/PanelSideBarContext";
+
+export interface PanelSideBarLayoutProps extends PropsWithChildren {
+  /**
+   * The brand content shown on the top navigation bar.
+   */
+  brand?: ReactNode;
+  /**
+   * The footer content.
+   */
+  footer?: ReactNode;
+  /**
+   * The collapsible option to choose.
+   */
+  collapsible?: boolean;
+}
+
+export const PanelSideBarLayout = (props: PanelSideBarLayoutProps) => {
+  const { brand, children, footer, collapsible = true } = props;
+
+  const [isOpen, setIsOpen] = useState(true);
+  const toggleSidebar = () => setIsOpen((prev) => !prev);
+
+  const {
+    brand: contextBrand,
+    footer: contextFooter,
+    userDropDownMenu,
+    userDropDownMenuToggle,
+    topBarCustomItems = [],
+  } = usePanelSideBarContext();
+
+  return (
+    <>
+      <nav id="nav-top" className="panel-layout navbar navbar-expand">
+        {/* Navbar Brand */}
+        <div className="navbar-brand">{brand ?? contextBrand}</div>
+        <Nav className="navbar-user" vertical style={{ marginRight: 48 + topBarCustomItems.length * 32 }}>
```

Comment:
the computation of margin right is wrong. In addition justify content set in the style is wrong. This has to look like


```suggestion
        <Nav className="navbar-user d-flex flex-row justify-content-end align-items-center" vertical style={{ marginRight: 48  }}>
```

Reply:
https://github.com/neolution-ch/react-pattern-ui/pull/42